web/angular-wallet/src/@fuse/components/shortcuts/shortcuts.component.ts

```diff
@@ -46,7 +46,7 @@ export class FuseShortcutsComponent implements OnInit, OnDestroy
         private _cookieService: CookieService,
         private _fuseMatchMediaService: FuseMatchMediaService,
         private _fuseNavigationService: FuseNavigationService,
-        private _observableMedia: ObservableMedia,
+        private _observableMedia: MediaObserver,
```

Comment:
OMG - These are kind of unnecessary breaking changes...why change name from ObservableMedia to MediaObserver?